database/migrations/2023_06_01_234438_create_events_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('events', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('name');
+            $table->text('comment');
+            $table->string('image');
+            $table->string('location');
+            $table->date('date');
+            $table->integer('status_id')->default(1); //1:available
```

Comment:
what are the possible values of the `status_id` field?

Reply:
1:available, 2:occupied, 3:done